### PR TITLE
spec: fix some typo in chinese translation

### PR DESCRIPTION
**Pull request by boliza**
Signed-off-by: Ranger Tsao <cao.zhifu@gmail.com>

fix some typo and one broken link

### CODE REVIEW

File: cloudevents/languages/zh-CN/spec.md [hunk 1]
```diff
@@ -270,22 +270,22 @@ CloudEvents 协议绑定或事件格式实现同样必须能够在规范字符
   - 必须是非空字符串
   - 应该以反向 DNS 名称作为前缀。该前缀域表明了定义此事件类型语义的组织。
 - 示例
-  - com.github.pull.create
-  - com.example.object.delete.v2
+  - com.github.pull_request.opened
```
Comment (duglin) [inline cloudevents/languages/zh-CN/spec.md:273]: seems like we should make this change in the English version too for consistency... if we want to change it

File: cloudevents/languages/zh-CN/spec.md [hunk 2]
```diff
@@ -270,22 +270,22 @@ CloudEvents 协议绑定或事件格式实现同样必须能够在规范字符
...
+  - com.example.object.deleted.v2
```
Comment (duglin) [inline cloudevents/languages/zh-CN/spec.md:274]: here too

**Comment by duglin** [comment]
@JieDing can you review? thanks

**Comment by JieDing** [comment]
> @JieDing can you review? thanks

Sure, I'll review it.

File: cloudevents/languages/zh-CN/spec.md (hunk 1 above)
Comment (JieDing) [reply cloudevents/languages/zh-CN/spec.md:273]: @duglin I checked the English version for these two lines, they've been changed already. 
https://github.com/cloudevents/spec/blob/676cef91678b86ae09fb39017588487690bfd42e/cloudevents/spec.md?plain=1#L350-L351

File: cloudevents/languages/zh-CN/spec.md (hunk 2 above)
Comment (JieDing) [reply cloudevents/languages/zh-CN/spec.md:274]: ditto the comments above

**Review by JieDing — COMMENTED** [review]
Thanks a ton for your careful check for the translation.  Here are some suggestions.

File: cloudevents/languages/zh-CN/spec.md [hunk 3]
```diff
@@ -270,22 +270,22 @@ CloudEvents 协议绑定或事件格式实现同样必须能够在规范字符
...
 
 
 #### datacontenttype/data内容类型
 
 - 类型: `String` [RFC 2046](https://tools.ietf.org/html/rfc2046)
-- 描述: `data` 值的内容类型。 此属性使`data`能够承载任何类型的内容，
+- 描述: `data` 值的内容类型。 此属性使 `data` 能够承载任何类型的内容，
   因此格式和编码可能与所选事件格式的不同。
-  例如，使用 [JSON envelope](../../formats/json-format.md#3-envelope)格式呈现的事件可能在数据中携带 XML 的payload，这个属性可以用来通知消费者
+  例如，使用 [JSON envelope](../../formats/json-format.md#3-envelope)格式呈现的事件可能在数据中携带 XML 的 payload，这个属性可以用来通知消费者
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/spec.md:286]:
```suggestion
  例如，使用 [JSON envelope](../../formats/json-format.md#3-envelope)格式呈现的事件可能在数据中携带 XML 的有效负载，这个属性可以用来通知消费者
```

File: cloudevents/languages/zh-CN/spec.md (hunk 1 above)
Comment (duglin) [reply cloudevents/languages/zh-CN/spec.md:273]: cool thanks!

**Comment by duglin** [comment]
@boliza did you want to make any of the suggested updates?

**Comment by boliza** [comment]
sorry for a bit busy these days. will update this pr doday

File: cloudevents/languages/zh-CN/spec.md (hunk 3 above)
Comment (boliza) [reply cloudevents/languages/zh-CN/spec.md:286]: OK.

**Comment by duglin** [comment]
@JieDing are you ok with the PR now?

**Comment by JieDing** [comment]
> @JieDing are you ok with the PR now?

@duglin  LGTM

**Comment by duglin** [comment]
Approved on the 9/15 call

**Pull request merged**